pagefaultgames/pokerogue
# [Move] Partial implementation of Stuff Cheeks

## What are the changes?

Almost fully implements Stuff Cheeks: User eats random berry and raises defense two stages. There is one aspect not implemented: "If the user is not holding a Berry, Stuff Cheeks cannot be selected from the move menu"
https://bulbapedia.bulbagarden.net/wiki/Stuff_Cheeks_(move)

## Why am I doing these changes?



I wanted to implement Stuff Cheeks to use it in my run, as it's currently completely unimplemented.

## What did change?


Stuff Cheeks becomes a partially implemented move and works almost completely as intended, minus the aspect of not being selectable when no berries are available.

### Screenshots/Videos


Stuff Cheeks with a berry:

https://github.com/pagefaultgames/pokerogue/assets/62030095/b94ce06f-c409-44b5-889f-8a3e9f45c3d1

Stuff Cheeks without a berry:

https://github.com/pagefaultgames/pokerogue/assets/62030095/17786447-bd6d-4acf-85c3-16d486db3069


## How to test the changes?




Use the overrides.ts file to start with the Stuff Cheeks move and add a berry to your starting held items. Use the move to see your berry be consumed and defense stat raised 2 stages.

## Checklist
